examples/run_neverworld_simulation.jl

```diff
@@ -0,0 +1,97 @@
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.TurbulenceClosures: CATKEVerticalDiffusivity
+using ClimaOcean.IdealizedSimulations: neverworld_simulation
+using Printf
+
+minimum_turbulent_kinetic_energy = 1e-6
+minimum_convective_buoyancy_flux = 1e-11
+closure = CATKEVerticalDiffusivity(; minimum_turbulent_kinetic_energy,
+                                   minimum_convective_buoyancy_flux)
+
+simulation = neverworld_simulation(GPU();
+                                   horizontal_size = (240, 280),
```

Comment:
We can do that. I used `size` because the number of grid points is straightforwardly related to performance (resolution has to be combined with the domain size to get performance). But I agree that resolution is probably the thing that you'd vary

Reply:
Done.